backend/otel/parse_test.go

```diff
@@ -0,0 +1,36 @@
+package otel
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func FuzzFormatStructureStackTrace(f *testing.F) {
+	f.Fuzz(func(t *testing.T, stackTrace string) {
+		formatStructureStackTrace(stackTrace)
+	})
+}
+
+func TestFormatStructureStackTrace(t *testing.T) {
+	var inputs = []struct {
+		language           string
+		stacktrace         string
+		expectedFrameError string
```

Comment:
What are we using this for? Should this be something we're asserting on down below?

Reply:
whoops! accidently removed the `expectedFrameError` check while testing. good catch!